timesyncd pulls in libm, only for one single exp2() in a debug log statement...
.. save the memory by not logging this little bit of info.


--- systemd-233/src/timesync/timesyncd-manager.c~	2017-03-01 21:43:06.000000000 +0000
+++ systemd-233/src/timesync/timesyncd-manager.c	2017-05-01 02:04:00.100245695 +0000
@@ -652,7 +652,7 @@
                   NTP_FIELD_VERSION(ntpmsg.field),
                   NTP_FIELD_MODE(ntpmsg.field),
                   ntpmsg.stratum,
-                  exp2(ntpmsg.precision), ntpmsg.precision,
+                  0, ntpmsg.precision,
                   root_distance,
                   ntpmsg.stratum == 1 ? ntpmsg.refid : "n/a",
                   origin - OFFSET_1900_1970,
